### PR TITLE
Rewrite Documentation, following sous-chefs.org guidelines

### PR DESCRIPTION
**Pull request by sinfomicien**
## Description

Rewrite documentations

### Issues Resolved

none.

### Check List
- [N/A] All tests pass. See https://github.com/sous-chefs/apache2/blob/master/TESTING.md
- [N/A] New functionality includes testing.
- [N/A] New functionality has been documented in the README if applicable

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This is a big Pull Request.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: documentation/resource_mariadb_client_install.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+## Actions
+
+- `install` - (default) Install client packages
+
+## Properties
+
+Name                | Types             | Description                                                   | Default                                   | Required?
+------------------- | ----------------- | ------------------------------------------------------------- | ----------------------------------------- | ---------
+`version`           | String            | Version of MariaDB to install                                 | '10.3'                                    | no
```
Comment (damacus) [inline documentation/resource_mariadb_client_install.md]:
```suggestion
`version`           | String            | Version of MariaDB to install                                 | `10.3`                                    | no
```

Comment (damacus) [reply documentation/resource_mariadb_client_install.md]: I think this stlye *might* look better when it's parsed.

**Comment by damacus** [comment]
Looks awesome! 

❤️  documentation

File: documentation/resource_mariadb_client_install.md (hunk 1 above)
Comment (sinfomicien) [reply documentation/resource_mariadb_client_install.md]: ok will adapt the whole documentation :-D

**Review by damacus — DISMISSED** [review]

**Pull request merged**